### PR TITLE
Refactor EVP - LRQA-83366 | Master

### PR DESCRIPTION
**Pull request by Italo-gabriel**
JIRA Ticket [LRQA-83366](https://liferay.atlassian.net/browse/LRQA-83366)

### CODE REVIEW

**Comment by Italo-gabriel** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase [hunk 1]
```diff
@@ -68,7 +68,7 @@ definition {
 				zipCode = "12.345-67");
 		}
 
-		task ("When the employee user create a request") {
+		task ("When the Employee user create a request") {
```
Comment (MGabriellaS) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase:71]: Just a typo here, could you change this task to: `"When the Employee user creates a request"`

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase [hunk 2]
```diff
@@ -82,7 +82,7 @@ definition {
 			EVPUtils.viewSuccessMessage();
 		}
 
-		task ("Then the employee user view a Request Created and also view Organization Created") {
+		task ("Then the Employee user view a Request Created and also view Organization Created") {
```
Comment (MGabriellaS) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase:85]: Just a typo here, could you change this task to: `"Then the Employee user can see the Request  and Organization Created on listing"`

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase [hunk 3]
```diff
@@ -100,7 +100,7 @@ definition {
 	@description = "LPS-162894 - Verify that is not possible to submit the form without fill all required fields"
 	@priority = 3
 	test CannotSubmitFormWithoutFillRequiredField {
-		task ("When the user submit an organization form without a required field filled in") {
+		task ("When the Employee user submit an organization form without a required field filled in") {
```
Comment (MGabriellaS) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase:103]: Just a typo here, could you change this task to: `"When the Employee user submits an organization form without a required field filled in"`

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase [hunk 4]
```diff
@@ -115,10 +115,10 @@ definition {
 		}
 	}
 
-	@description = "LPS-162894 - Verify that after submit the form, when click on 'Done' button, will be redirected back to 'Organization' page"
+	@description = "LPS-162894 - Verify that after submit the form, when click on 'Done' button, will view a 'Thank you' message and be redirected back to 'Organization' page"
 	@priority = 2
 	test CanSeeRedirectionToOrganizationPageAfterCreateOrganization {
-		task ("When the employee user create an organization") {
+		task ("When the Employee user create an organization") {
```
Comment (MGabriellaS) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase:121]: Just a typo here, could you change this task to: `"When the Employee user creates an organization"`

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase [hunk 5]
```diff
@@ -138,38 +138,10 @@ definition {
...
-				organizationSite = "www.test.com.br",
-				smallDescription = "Description Test",
-				state = "State Test",
-				street = "Street Test",
-				taxId = 12345,
-				zipCode = "12.345-67");
-		}
-
-		task ("Then can see the message of successfully is displayed") {
+		task ("Then The Employee user view a 'Thank you' message and is Redirection to page Organization") {
```
Comment (MGabriellaS) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase:141]: Just a typo here, could you change this task to: `"Then The Employee user view a 'Thank you' message and is Redirected to page Organization"`

**Review by MGabriellaS — CHANGES_REQUESTED** [review]
Thanks @Italo-gabriel  for work on this!
Please take a look on these comments below!
cc: @vinicius-paes

**Review by MGabriellaS — COMMENTED** [review]
Thanks @Italo-gabriel  for working on this!
LGTM!
cc: @vinicius-paes

**Review by vinicius-paes — APPROVED** [review]

**Comment by vinicius-paes** [comment]
ci:forward

**Pull request closed without merging**